Uchu.StandardScripts/General/UnusedScript.cs

```diff
@@ -0,0 +1,23 @@
+using Uchu.World;
+using Uchu.World.Scripting.Native;
+
+namespace Uchu.StandardScripts.General
+{
+    //done with one script, comment out if needed
+    [ScriptName("l_qb_spawner.lua")]
+    //already implemented by checking if an object has the POI tag on every collision
+    [ScriptName("l_poi_mission.lua")]
+    //function done by LaunchpadEvent.cs
+    [ScriptName("l_ag_zone_player.lua")]
+    public class UnusedScript : ObjectScript
+    {
+        /// <summary>
+        /// Creates the object script.
+        /// </summary>
+        /// <param name="gameObject">Game object to control with the script.</param>
+        public UnusedScript(GameObject gameObject) : base(gameObject)
+        {
+            //TODO: replace this with actual script ignore list
```

Comment:
Not a fan of shipping a feature like this as opposed to a proper whitelist. However, I am not sure if this requires changes in 1 place or 2 places because scripts have 2 places they can load from.

Reply:
You could just umm leave it not implemented?

Reply:
I recall this being implemented to reduce the output for no script implementations. This solution is too hacky in my opinion compared to not having this or having an ignore list of scripts.

Reply:
It was indeed added to reduce log output.